packages/aws-cdk-lib/aws-globalaccelerator/lib/accelerator.ts

```diff
@@ -21,6 +21,22 @@ export interface IAccelerator extends cdk.IResource {
    * @attribute
    */
   readonly dnsName: string;
+
+  /**
+   * IP addressess associated with the accelerator.
+   *
+   * @attribute
+   */
+  readonly ipAddresses?: string[];
+
+  /**
+   * The IP address type that the accelerator supports.
+   *
+   * For a standard accelerator, the value can be IPV4 or DUAL_STACK.
+   *
+   * @attribute
+   */
+  readonly ipAddressType?: IpAddressType;
```

Comment:
Depending on [this comment](https://github.com/aws/aws-cdk/pull/28055/files#r1400194833), it will be removed.